cloudsql/postgresql/server.js

```diff
@@ -0,0 +1,241 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// Require process, so we can mock environment variables
+const process = require('process');
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const Knex = require('knex');
+
+const app = express();
+app.set('view engine', 'pug');
+app.enable('trust proxy');
+
+// Automatically parse request body as form data
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+
+// Set Content-Type for all responses for these routes
+app.use((req, res, next) => {
+  res.set('Content-Type', 'text/html');
+  next();
+});
+
+// Create a Winston logger that streams to Stackdriver Logging
+const winston = require('winston');
```

Comment:
Do we need `winston` in a postgres sample?

Reply:
The canonical sample in Java demonstrates logging which is why Winston is included here.